loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -122,8 +122,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		this._toolitemHandlers['.uno:FrameLineColor'] = this._colorControl;
 		this._toolitemHandlers['.uno:Color'] = this._colorControl;
 		this._toolitemHandlers['.uno:FillColor'] = this._colorControl;
-		this._toolitemHandlers['.uno:ResetAttributes'] = this._clearFormattingControl;
-		this._toolitemHandlers['.uno:SetDefault'] = this._clearFormattingControl;
+		this._toolitemHandlers['.uno:ResetAttributes'] = this._formattingControl;
```

Comment:
now both clone and clear direct format seem to have the same id : 
![image](https://user-images.githubusercontent.com/65948705/119089667-cd325800-ba0a-11eb-948f-b6f4405ad8dd.png)